ts/src/pro/bitfinex2.ts

```diff
@@ -269,7 +269,7 @@ export default class bitfinex2 extends bitfinex2Rest {
         //
         const name = 'myTrade';
         const data = this.safeValue (message, 2);
-        const trade = this.parseWsTrade (data, false);
+        const trade = this.parseWsTrade (data);
```

Comment:
@samgermain weren't they relying on this `false` argument? if so don't know if we can simply remove it. If they need a custom signature they we can always rename it to something like `parseWsOrderCustom` or smth like that

Reply:
I handle the logic a different way, look at the changes made to bitfinex2.parseWsTrade we can't unify the method if we don't remove it

Reply:
> @samgermain weren't they relying on this `false` argument? if so don't know if we can simply remove it. If they need a custom signature they we can always rename it to something like `parseWsOrderCustom` or smth like that

I changed the implementation of bitfinex2.parseWsOrder so that the argument is no longer required